canvasapi/assignment.py

```diff
@@ -160,6 +160,28 @@ def get_peer_reviews(self, **kwargs):
             _kwargs=combine_kwargs(**kwargs),
         )
 
+    def get_provisional_grades_status(self, **kwargs):
+        """
+        Tell whether the student's submission needs one or more provisional grades.
+
+        :calls: `GET /api/v1/courses:course_id/provisional_grades/status \
```

Comment:
URL doesn't match the Canvas docs or the one you're actually calling

```suggestion
        :calls: `GET /api/v1/courses/:course_id/assignments/:assignment_id/provisional_grades/status \
```